docs/site/Parsing-requests.md

```diff
@@ -0,0 +1,159 @@
+---
+lang: en
+title: 'Parsing requests'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Parsing-requests.html
+summary:
+---
+
+## Parsing Requests
+
+This is an action in the default HTTP sequence, it parses arguments from an
```

Comment:
Can you specify what the key that the action is bound to is (PARSE_PARAMS)? It might be good to see it in code as well

Reply:
I am not sure what's the point of introducing the binding key here...?

Reply:
The coercion/validation is done by `parser.ts`'s `parseOperationArgs`, which is bound by default to the key `RestBindings.SequenceActions.PARSE_PARAMS`. Without this action being triggered int he sequence, no validation/coercion would be done.

The section is also a child page of `Sequence.md`, so might as well

Reply:
I see...my doc only covers the default behaviour of this action, and I think those actions would be bound automatically after extending `Application` with the rest mixin :)

I would like to add a section on `sequence` for replacing/customizing the actions, and list the binding key for each action under it, if such a section doesn't exist. Otherwise we need to repeat these things in every action: in which situations they come out of the box, in which situations 
it's on the users to bind them manually, and how to replace them with a custom action.

thought?